modular/manager/bucket_migrate_scheduler.go

```
@@ -550,11 +564,20 @@ func (s *BucketMigrateScheduler) getExecutePlanByBucketID(bucketID uint64) (*Buc
 	if ok {
 		return executePlan, nil
 	} else {
-		// TODO
 		return nil, errors.New("no such execute plan")
 	}
 }
 
+func (s *BucketMigrateScheduler) deleteExecutePlanByBucketID(bucketID uint64) error {
+	_, ok := s.executePlanIDMap[bucketID]
+	if ok {
+		delete(s.executePlanIDMap, bucketID)
+		return nil
+	} else {
+		return errors.New("no such execute plan")
```

It's better to ignore this err?